gen/build_deploy/aws.py

```diff
@@ -485,10 +485,7 @@ def gen_templates(arguments):
     with logger.scope("Validating CloudFormation"):
         validate_cf(cloudformation)
 
-    return gen.Bunch({
-        'cloudformation': cloudformation,
-        'results': results
-    })
+    return ResultTuple(cloudformation, results)
```

Comment:
I like using `.` to lookup guaranteed members rather than doing the dictionary indexing. `_as_artifact_and_pkg` also currently requires the API of having the `.` syntax, which we can change but I'd like to not pull into this PR